data/data_fixes/fix_sample_ids.rb

```diff
@@ -0,0 +1,34 @@
+require 'csv'
+require 'logger'
+
+ActiveRecord::Base.logger = nil
+logger = Logger.new("sample_ids_log.log")
+data = CSV.parse(File.read("data/data_fixes/root_sample_ids.csv"), headers: true)
+failed_ids = Array.new()
+success_ids = Array.new()
+
+data.each_with_index do |row, index|
+    if index % 1000 == 0
+        puts "."
+    end
+    samples = Sample::Metadata.where("sample_description" => row["original_root_sample_id"])
+    if samples != []
+        samples.each do |sample|
+            puts "updating: #{sample.sample_description}"
+            sample.sample_description = row["root_sample_id"]
+            saved = sample.save
+            if saved
+                success_ids << row["original_root_sample_id"]
+            else
+                failed_ids << row["original_root_sample_id"]
+            end
```

Comment:
I think it would make more sense to store the id of the sample_metadata where the update failed; if not we would have a list that will contain several times the same root_sample_id without indication of which specific sample failed to update